docs/usage/configuration-options.md

```diff
@@ -4041,7 +4041,7 @@ This limit is enforced on a per-repository basis.
 This setting tells Renovate _when_ to create PRs:
 
 - `immediate` (default): Renovate creates PRs immediately after creating the corresponding branch
-- `not-pending`: Renovate waits until status checks have completed (passed or failed) before raising the PR
+- `not-pending`: Renovate waits until status checks have completed (passed or failed) before raising the PR (or creating the branch)
```

Comment:
🤔 are you sure? don't mix it up with strict upgrade filtering which doesn't take commit checks into account.